Bayes_Rules/Chapter 4 Balance and Sequentiality in Bayesian Analyses.ipynb

```diff
@@ -0,0 +1,818 @@
+{
```

Comment:
<p>IIs something supposed to be here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc_resources/pull/188/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>